lib/filewatch/bootstrap.rb

```diff
@@ -0,0 +1,91 @@
+# encoding: utf-8
+require "rbconfig"
+require "pathname"
+
+## Common setup
+#  all the required constants and files
+#  defined in one place
+module FileWatch
+  HOST_OS_WINDOWS = (RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/) != nil
+  # the number of bytes read from a file during the read phase
+  FILE_READ_SIZE = 32768
+  # the largest fixnum in ruby
+  # this is used in the read loop e.g.
+  # @opts[:read_iterations].times do
+  # where read_iterations defaults to this constant
+  FIXNUM_MAX = (2**(0.size * 8 - 2) - 1)
+
+  require_relative "helper"
+
+  if HOST_OS_WINDOWS
+    require "winhelper"
+    FILEWATCH_INODE_METHOD = :win_inode
```

Comment:
suggestion: like we don in LS for the `windows?` method instead of using a constant, maybe it would be cleaner to define methods likes `win_inodes?` and `nix_inodes?`